.vortex/tooling/tests/Traits/MockTrait.php

```
@@ -0,0 +1,780 @@
+        // Sort both arrays for consistent comparison.
+        sort($expected_headers);
+        sort($actual_headers);
+
+        if ($expected_headers !== $actual_headers) {
+          throw new \RuntimeException(sprintf('mail() called with unexpected headers. Expected "%s", got "%s".', print_r($expected_headers, TRUE), print_r($actual_headers, TRUE)));
+        }
+      }
+
+      return $response['result'];
+    });
+  }
```

PHPMD (2.15.0)

[warning] The method mockMailMultiple() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)